Xext/xres.c

```diff
@@ -273,12 +273,28 @@ ProcXResQueryClientResources(ClientPtr client)
 
     FindAllClientResources(resClient, ResFindAllRes, counts);
 
+    xXResType *scratch = calloc(sizeof(xXResType), lastResourceType + 1);
```

Comment:
It probably doesn’t matter, but the order of calloc arguments should be reversed to follow the spec.

Reply:
fixed.